gtsfm/averaging/rotation/shonan.py

```diff
@@ -71,12 +72,17 @@ def __between_factors_from_2view_relative_rotations(
         return between_factors
 
     def _between_factors_from_pose_priors(
-        self, i2Ti1_priors: Dict[Tuple[int, int], PosePrior], old_to_new_idxs: Dict[int, int]
+        self, relative_pose_priors: Dict[Tuple[int, int], PosePrior], old_to_new_idxs: Dict[int, int]
     ) -> BetweenFactorPose3s:
         """Create between factors from the priors on relative poses."""
         between_factors = BetweenFactorPose3s()
+        noise_model = gtsam.noiseModel.Isotropic.Sigma(POSE3_DOF, PRIOR_SIGMA)
+        # temporary fix
```

Comment:
@dellaert is this fix ok temporarily?

Context: we need Isotropic noise model for shonan, but pose-slam gives us Gassian models.

Reply:
Sure but add a TODO: take average covariance instead